res/controllers/Reloop-Jockey-3-ME-scripts.js

```diff
@@ -0,0 +1,444 @@
+Jockey3ME.hotcueClearVal = 0;
+Jockey3ME.crossfaderScratch = false;
+Jockey3ME.effectSelectTimer = 0;
+Jockey3ME.num_effectsValue = [0,0,0,0];
+Jockey3ME.effectsAvailable = 5; // Sets how many Effects are Loadable
+Jockey3ME.move_beat_value = 4; // Sets how many Beats Jumping when "MOVE" is Turned
+Jockey3ME.CUP_value = 0;
+Jockey3ME.MixerDeck1 = 0;
+Jockey3ME.MixerDeck2 = 0;
+Jockey3ME.noVolHopValue = [false,false,false,false,false];
+
+// Functions
+Jockey3ME.EffectLedMeterShow = function () {
+  midi.sendShortMsg(0x90,0x1D,Jockey3ME.EffectLedMeterValue);
+  midi.sendShortMsg(0x91,0x1D,Jockey3ME.EffectLedMeterValue);
+  Jockey3ME.EffectLedMeterValue += 2;
+  if (Jockey3ME.EffectLedMeterValue >= 127) {
+    engine.stopTimer(Jockey3ME.EffectLedMeter);
+    Jockey3ME.EffectLedMeter = 0;
+
+    // Sets Effect Leds
+    for (var i = 1, j = 176; i <= 4; i++) {
+      Jockey3ME.effectSelectLedSet(j,i);
+      j++;
+    }
+  };
+}
+// Main LedShow Script
+Jockey3ME.LedMeterShow = function() {
+  midi.sendShortMsg(0x90,0x21,Jockey3ME.LedMeterShowValue);
+  midi.sendShortMsg(0x91,0x21,Jockey3ME.LedMeterShowValue);
+  if (Jockey3ME.LedMeterShowValueTwo == false) {
+    ++Jockey3ME.LedMeterShowValue;
+  } else {
+    --Jockey3ME.LedMeterShowValue;
+  }
+  if (Jockey3ME.LedMeterShowValue > 10) Jockey3ME.LedMeterShowValueTwo = true;
+  // Stop The LedShow and Start Scanning VuMeter
+  if (Jockey3ME.LedMeterShowValueTwo && Jockey3ME.LedMeterShowValue <= 0) {
+    engine.stopTimer(Jockey3ME.LedMeterShowTimer);
+    Jockey3ME.LedMeterShowTimer = 0;
+    Jockey3ME.VuMeter = engine.beginTimer(20,"Jockey3ME.fVuMeter()"); // Start Every 20ms the fVuMeter Function
+    Jockey3ME.EffectLedMeter = engine.beginTimer(20,"Jockey3ME.EffectLedMeterShow()");
+  };
+}
+
+Jockey3ME.LedShowBegin = function () {
+  Jockey3ME.LedMeterShowTimer = engine.beginTimer(40,"Jockey3ME.LedMeterShow()");
+}
+
+// Init Script at Programmstart
+Jockey3ME.init = function () {
+  for (var i = 1; i < 120; i++) {
+    midi.sendShortMsg(0x90,i,0x7F);
+    midi.sendShortMsg(0x91,i,0x7F);
+    midi.sendShortMsg(0x92,i,0x7F);
+    midi.sendShortMsg(0x93,i,0x7F);
+  };
+  for (var j = 1; j < 120; j++) {
+    midi.sendShortMsg(0x90,j,0x00);
+    midi.sendShortMsg(0x91,j,0x00);
+    midi.sendShortMsg(0x92,j,0x00);
+    midi.sendShortMsg(0x93,j,0x00);
+  };
+  Jockey3ME.LedShowBeginTimer = engine.beginTimer(500,"Jockey3ME.LedShowBegin()",1); // LedShow Script Starts Here after 500ms
+}
+
+// Sets VuMeter new Value to Leds
+Jockey3ME.fVuMeter = function () {
+  var VuVal1 = engine.getValue("[Channel1]","VuMeter");
+  var VuVal2 = engine.getValue("[Channel2]","VuMeter");
+  var VuVal3 = engine.getValue("[Channel3]","VuMeter");
+  var VuVal4 = engine.getValue("[Channel4]","VuMeter");
+  VuVal1 = VuVal1 * 10;
+  VuVal1 = parseInt(VuVal1);
+  VuVal2 = VuVal2 * 10;
+  VuVal2 = parseInt(VuVal2);
+  VuVal3 = VuVal3 * 10;
+  VuVal3 = parseInt(VuVal3);
+  VuVal4 = VuVal4 * 10;
+  VuVal4 = parseInt(VuVal4);
```

Comment:
This can all be done in the variable definition:
``var VuVal1 = parseInt(engine.getValue("[Channel1]","VuMeter") * 10);``